### PR TITLE
Feature/add support plan management

### PR DESCRIPTION
**Pull request by wvanheerde**
Added support for retrieving + updating the current support plan

### CODE REVIEW

File: coto/clients/signin_aws/__init__.py [hunk 1]
```diff
@@ -166,12 +166,14 @@ def mfa_required(self, email):
         mfa_client = self.session().client('mfa')
         mfa = mfa_client.get_mfa_status(email)
         if 'mfaType' in mfa:
-            if mfa['mfaType'] != 'SW':
-                raise Exception("cannot handle hardware mfa tokens")
+            if mfa['mfaType'] == 'NONE':
```
Comment (tader) [inline coto/clients/signin_aws/__init__.py]: This change removes the check for 'SW'!

File: coto/clients/federation.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
 from furl import furl
 import json
-import requests
+from botocore.vendored import requests
```
Comment (tader) [inline coto/clients/federation.py]: Without botocore in the Pipfile it fails the tests...

File: coto/clients/support.py [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+        if 'X-CSRF-Token' in r.headers:
+            self.__xsrf_token = r.headers['X-CSRF-TOKEN']
+
+        if r.status_code != 200:
+            raise Exception("failed post {0}".format(api))
+
+        return json.loads(r.text)
+
+    def get_support_level(self):
```
Comment (tader) [inline coto/clients/support.py]: Documentation is missing, see the billing Client for a nice example.

File: coto/clients/signin_aws/__init__.py (hunk 1 above)
Comment (wvanheerde) [reply coto/clients/signin_aws/__init__.py]: It just checks whether MFA is enabled at all. Turned out AWS is not always reliably returning the mfaType. So far we have seen 'NONE', 'SW' or just '', which also means at least some form of MFA is enabled. This is the result of trying to fix the MFA issues we were having back then, but be my guest if you have a better suggestion.

**Pull request merged**